<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="1" column="0">
     <widget class="QTextEdit" name="tokenInput">
      <property name="placeholderText">
       <string>Paste Token</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="startDateLabel">
      <property name="text">
       <string>Start Date</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QDateEdit" name="startDatePicker">
      <property name="minimumDate">
       <date>
        <year>1900</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
      <property name="date">
       <date>
        <year>2014</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="endDateLabel">
      <property name="text">
       <string>End Date</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QDateEdit" name="endDatePicker">
      <property name="minimumDate">
       <date>
        <year>1900</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
      <property name="date">
       <date>
        <year>2014</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="getButton">
      <property name="text">
       <string>Get Product Details</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="HeaderLabel">
      <property name="text">
       <string>eBay Details Getter</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>821</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
